src/containers/my-quizzes/QuizzesContainer.tsx

```diff
@@ -108,25 +114,18 @@ const TestsContainer = () => {
     }
   ]
 
-  return (
-    <Box>
-      {loading ? (
-        <Loader pageLoad size={50} />
-      ) : (
-        <EnhancedTable
-          columns={columnsToShow}
-          data={{ items: response.items }}
-          emptyTableKey='myResourcesPage.quizzes.emptyQuizzes'
-          rowActions={rowActions}
-          sort={{
-            sort: { order: SortEnum.Desc, orderBy: 'updatedAt' },
-            onRequestSort: () => null
-          }}
-          sx={styles.table}
-        />
-      )}
-    </Box>
+  const tableAttachments = (
```

Comment:
I think it's not table of attachments :)

Reply:
> I think it's not table of attachments :)

You are absolutely right, i missed that, thank you, fixed.